src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -76,6 +77,14 @@ public void setNotes(List<Note> notes) {
         for (Note note: notes) {
             if (note.isClientNote()) {
                 // handle client notes
+                // todo:  =======================================================
+                //        decided to not store the client notes in a separete set
+                //        because the notes are going to be stored within clients
+                //        itself. The setClient function aldy does this. Adding in
+                //        a collection of client notes in this class would mean that
+                //        collection needs to be constantly updated. As such, it
+                //        might be good enough to just modify the getNoteList method
+                //        ===========================================================
```

Comment:
@raysonkoh I'm sticking to this until it comes off as a problem. 

(for my own reference, relevant alernative: [this commit for a deleted PR](https://github.com/rtshkmr/tp/commit/e30108c552aad0b342cb65e5bc39af07308b50c3) )